hyp3_testing/helpers.py

```diff
@@ -0,0 +1,93 @@
+import os
+import json
+import random
+import string
+from glob import glob
+from pathlib import Path
+from zipfile import ZipFile
+
+import requests
+from hyp3lib.fetch import download_file
+from jinja2 import Template
+
+from hyp3_testing import AUTH_URL
+
+
+def hyp3_session(username: str = None, password: str = None):
+    username = username if username is not None else os.environ.get('HYP3_USERNAME')
+    password = password if password is not None else os.environ.get('HYP3_PASSWORD')
+    if username is None and password is None:
+        auth = None
+    else:
+        auth = (username, password)
+
+    session = requests.Session()
+    resp = session.get(AUTH_URL, auth=auth)
+    resp.raise_for_status()
+
+    return session
+
+
+def get_submission_payload(template):
+    with open(template) as f:
+        body = f.read()
+
+    name = ''.join(random.choices(string.ascii_letters + string.digits, k=7))
+    jobs = Template(body).render(name=name)
+
+    return json.loads(jobs)
+
+
+def get_jobs_update(name, url, session, request_time=None):
+    params = {'name': name}
+    if request_time is not None:
+        params.update({'start': request_time, 'end': request_time})
+
+    update = session.get(url, params=params)
+    update.raise_for_status()
+    return update
```

Comment:
should `return update.json()` so the call doesn't have to mess with a `response` object